Jenkinsfile

```diff
@@ -37,23 +37,25 @@ spec:
       deploymentSize: 1
       web:
         enabled: false
-  elasticsearch_manifest: |
-    apiVersion: elasticsearch.k8s.elastic.co/v1beta1
+  elasticsearchManifest: |
+    apiVersion: elasticsearch.k8s.elastic.co/v1
     kind: Elasticsearch
     metadata:
       name: elasticsearch
       namespace: $namespace
     spec:
       version: 7.10.2
+      volumeClaimDeletePolicy: DeleteOnScaledownAndClusterDeletion
       http:
         tls:
           certificate:
             secretName: 'elasticsearch-es-cert'
       nodeSets:
         - config:
-            node.data: true
-            node.ingest: true
-            node.master: true
+            node.roles:
+              - master
+              - data
+              - ingest
             node.store.allow_mmap: true
```

Comment:
Did you look up if this keyname changed too?

Reply:
I did not, but I'll see if I can figure it out. I think this is a free-form value for ElasticSearch configuration... and if that's the case, then it shouldn't have changed as the deployed ElasticSearch version is locked at 7.10.2 currently.

Reply:
https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-virtual-memory.html

> By default, Elasticsearch uses memory mapping (mmap) to efficiently access indices. Usually, default values for virtual address space on Linux distributions are too low for Elasticsearch to work properly, which may result in out-of-memory exceptions. This is why the quickstart example disables mmap via the node.store.allow_mmap: false setting. For production workloads, it is strongly recommended to increase the kernel setting vm.max_map_count to 262144 and leave node.store.allow_mmap unset.

In our case OpenShift has the appropriate labels set up to automatically set the max_map_count to an appropriate value for ElasticSearch, so I think setting this to `true` is appropriate here.